client/src/components/Ticketing/event/TicketPicker.tsx

```diff
@@ -392,12 +392,11 @@ const TicketPicker = (props: TicketPickerProps): ReactElement => {
               className='text-zinc-300'
               value=''
               disabled
```

Comment:
Thanks for fixing all of the console errors in this file!! They were bothering me but I didn't exactly know which things to remove